src/region/__tests__/RegionRect-test.tsx

```diff
@@ -8,22 +8,32 @@ jest.mock('../regionUtil', () => ({
 }));
 
 describe('RegionRect', () => {
-    const getWrapper = (props = {}): ShallowWrapper => shallow(<RegionRect {...props} />);
+    const getWrapper = (props = {}): ReactWrapper => mount(<RegionRect {...props} />);
```

Comment:
This will definitely slow down the test suites a bit, so it's best to leave it shallow if possible.